mapping/sustainability.yaml

```diff
@@ -381,7 +381,7 @@
     title: Maintenance plan or program
     module: Economic and fiscal
     indicator: Economic viability
-    disclosure format: Disclose the maintenance plan or program.
+    disclosure format: Disclose the maintenance plan or program. This is the documentation that describes work to prevent the breakdown or malfunctioning of an asset.
```

Comment:
Should we wait until https://github.com/open-contracting/infrastructure/issues/237 is resolved and then align the wording here with the definitions of maintenance in the project type and project status codelists?

Reply:
yes that's probably sensible